packages/is-shallow-equal/index.js

```diff
@@ -30,3 +30,5 @@ function isShallowEqual( a, b ) {
 }
 
 module.exports = isShallowEqual;
+module.exports.isShallowEqualObjects = isShallowEqualObjects;
+module.exports.isShallowEqualArrays = isShallowEqualArrays;
```

Comment:
I'm probably missing something but shouldn't all this calls return false?
```
wp.isShallowEqual.isShallowEqualArrays( [ 1, 2 ] [3, 4 ] );
true
wp.isShallowEqual.isShallowEqualArrays( [ 1, 2 ] [3, 4 ] );
true
wp.isShallowEqual.isShallowEqualArrays( [ 1, 2 ] [1, 2, 3 ] );
true
wp.isShallowEqual( [ 1, 2 ] [1, 2, 3 ] );
true
```

This PR is not changing anything on the existing shallow compare logic previously if we used isShallowEqual instead of isShallowEqualArrays the result was the same.

Reply:
I think this can be attributed to a missing comma. I'm actually surprised it's not a syntax error.

```js
wp.isShallowEqual.isShallowEqualArrays( [ 1, 2 ] [ 3, 4 ] );
// true

wp.isShallowEqual.isShallowEqualArrays( [ 1, 2 ], [ 3, 4 ] );
// false
```

Reply:
This would be a member access:

```
[ 'foo', 'bar' ][ 0 ]
// 'foo'
```

It's close to what you've written, but I'm not sure what would be expected (and why it doesn' throw) when passed _multiple_ indices.

Reply:
@jorgefilipecosta Could you verify whether the above explains what you'd seen?